contingency/contingency-dsl/src/test/resources/META-INF/services/com.powsybl.contingency.dsl.ExtendableDslExtension

```diff
@@ -0,0 +1 @@
+com.powsybl.contingency.dsl.ProbabilityContingencyDslExtension
```

Comment:
I think this files is not necessary with the `@AutoService` extension ?

Reply:
actually @AutoService didn't work so I put it manually, I guess I forgot to remove the @AutoService annotation

Reply:
I remove this file and after a `mvn clean` it works. I think you remove it.